treadmill/infra/constants.py

```diff
@@ -20,4 +20,12 @@
 
     }
 }
+ROLES = {
+    'MASTER': 'MASTER',
+    'NODE': 'NODE',
+    'LDAP': 'LDAP',
+    'IPA': 'IPA',
+    'ZOOKEEPER': 'ZOOKEEPER',
+    'DEFAULT': 'DEFAULT'
+}
 DEFAULT_DOMAIN = 'treadmill.org'
```

Comment:
on a second thought, i would make it required argument and not have it in constants. it really does not belong, easy to specify (I doubt it is correct value for anyone) and we do not really own treadmill.org unfortunately.